keras_cv/layers/object_detection/roi_sampler_test.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.object_detection.roi_sampler import _ROISampler
+from keras_cv.ops.box_matcher import ArgmaxBoxMatcher
+
+
+class ROISamplerTest(tf.test.TestCase):
+    def test_roi_sampler(self):
+        box_matcher = ArgmaxBoxMatcher([0.3], [-1, 1])
```

Comment:
let's add the param here to make it more readable. Same for _ROISampler below.

Reply:
Done.